airbyte-integrations/connectors/source-ebay-fulfillment/manifest.yaml

```diff
@@ -0,0 +1,1033 @@
+version: 6.48.12
+
+type: DeclarativeSource
+
+description: >-
+  Website: https://www.ebay.com/
+
+  Documentation:
+  https://developer.ebay.com/api-docs/sell/fulfillment/overview.html
+
+check:
+  type: CheckStream
+  stream_names:
+    - orders
+
+definitions:
+  streams:
+    orders:
+      type: DeclarativeStream
+      name: orders
+      primary_key:
+        - orderId
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /sell/fulfillment/v1/order
+          http_method: GET
+          error_handler:
+            type: CompositeErrorHandler
+            error_handlers:
+              - type: DefaultErrorHandler
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - orders
+          schema_normalization: Default
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            field_name: offset
+            inject_into: request_parameter
+          page_size_option:
+            type: RequestOption
+            field_name: limit
+            inject_into: request_parameter
+          pagination_strategy:
+            type: OffsetIncrement
+            page_size: 200
```

Comment:
`200` is the max page size allowed by the orders API: https://developer.ebay.com/api-docs/sell/fulfillment/resources/order/methods/getOrders